virttest/qemu_devices/qcontainer.py

```diff
@@ -2659,6 +2680,12 @@ def pcic_by_params(self, name, params, parent_bus=None):
                                 parent_bus=parent_bus,
                                 child_bus=bus)
 
+    def throttle_group_define_by_params(self, params, name):
+        throttle_group_parameter = params.get(
```

Comment:
refer to images_define_by_params, we pass image_params as the argument, i.e. we get image_params by params.object_params(tag) already. Here if params is not group-specified params(params.object_params(group)), we might have to get group-specified params first, for throttle_group_parameter_<groupid> might not be defined, we can define a throttle_group_parameter for all groups